diff --git a/captures.json b/captures.json
index 76bcd2e..b634081 100644
--- a/captures.json
+++ b/captures.json
@@ -1,6 +1,6 @@
 {
   "schema": 5,
-  "vpsadmin_commit": "b4ef8535a629ee8c9cd753afecdb05e0895d0eea",
+  "vpsadmin_commit": "6b3628af665049dc095ba985ef0fbe8a22286863",
   "assets": [
     {
       "id": "getting-started/members-list",
diff --git a/contract/navigation.yml b/contract/navigation.yml
index 9cd0938..d0332a8 100644
--- a/contract/navigation.yml
+++ b/contract/navigation.yml
@@ -1,5 +1,5 @@
 schema: 1
-vpsadmin_revision: b4ef8535a629ee8c9cd753afecdb05e0895d0eea
+vpsadmin_revision: 6b3628af665049dc095ba985ef0fbe8a22286863
 languages:
   - cs
   - en
diff --git a/contract/pages.yml b/contract/pages.yml
index d73ae73..f81d5e4 100644
--- a/contract/pages.yml
+++ b/contract/pages.yml
@@ -2,7 +2,7 @@
 schema: 2
 repository: vpsfreecz/vpsfree-kb-contracts
 revisions:
-  vpsadmin: b4ef8535a629ee8c9cd753afecdb05e0895d0eea
+  vpsadmin: 6b3628af665049dc095ba985ef0fbe8a22286863
   vpsadminos: 6bdf458fd9105379860234ff33d352e55844f08f
 pages:
   firewall:
diff --git a/flake.lock b/flake.lock
index 617c24a..1d4abda 100644
--- a/flake.lock
+++ b/flake.lock
@@ -164,17 +164,17 @@
         "vpsadminos": "vpsadminos"
       },
       "locked": {
-        "lastModified": 1791195217,
-        "narHash": "sha256-0T5vsIj7zDrqBSS40xLyQ4/NGL4+M4uxBgZiuCo+jJE=",
+        "lastModified": 1791292241,
+        "narHash": "sha256-CSZ+bZOaYQjU29bQ6S7AsbtcrYXcf/8mXrRuTDar1Ak=",
         "owner": "vpsfreecz",
         "repo": "vpsadmin",
-        "rev": "b4ef8535a629ee8c9cd753afecdb05e0895d0eea",
+        "rev": "6b3628af665049dc095ba985ef0fbe8a22286863",
         "type": "github"
       },
       "original": {
         "owner": "vpsfreecz",
         "repo": "vpsadmin",
-        "rev": "b4ef8535a629ee8c9cd753afecdb05e0895d0eea",
+        "rev": "6b3628af665049dc095ba985ef0fbe8a22286863",
         "type": "github"
       }
     },
diff --git a/flake.nix b/flake.nix
index 4dfdc6e..81a2985 100644
--- a/flake.nix
+++ b/flake.nix
@@ -2,7 +2,7 @@
   description = "Reproducible contracts for selected vpsFree.cz KB pages";
 
   inputs = {
-    vpsadmin.url = "github:vpsfreecz/vpsadmin/b4ef8535a629ee8c9cd753afecdb05e0895d0eea";
+    vpsadmin.url = "github:vpsfreecz/vpsadmin/6b3628af665049dc095ba985ef0fbe8a22286863";
     vpsadmin.inputs.vpsadminos.url = "github:vpsfreecz/vpsadminos/6bdf458fd9105379860234ff33d352e55844f08f";
     vpsadminos.follows = "vpsadmin/vpsadminos";
     vpsfStatus = {
